### ISSUE TITLE
Small improvements

### ISSUE THREAD

**Issue by MareenG** [gnosis/gp-swap-ui#278]
I am coming across a few inconsistencies and things that distract me. 
It is nothing major and my personal opinion.
I already opened two issues  (#276 and #277)  but from now on would gather those things in this issue and not spam you with individual issues.

- In the header it says **connect to a wallet** and in the body part it says **connect wallet**
- In light mode "web and contracts" link  and the block link  are barely visible
- in the explorer the owl in the header spins if you click on it would be cool to have the same for swap

**Comment by MareenG** [comment]
If I change the swapping tokens e.g. from DAI to USDC without trading the value which was inserted before remains in the **From** input.
I think it might be nicer to clear the sections when choosing a new token

https://user-images.githubusercontent.com/34510341/111623670-f526f480-87ea-11eb-9115-2a5e4bf94118.mov

**Comment by MareenG** [comment]
Custom slippage%: 
I think it might make sense to limit the decimal input only allowing to place two positions after the decimal point.

**Comment by anxolin** [comment]
> I am coming across a few inconsistencies and things that distract me.
It is nothing major and my personal opinion.
I already opened two issues (#276 and #277) but from now on would gather those things in this issue and not spam you with individual issues.

Thanks for not spamming. Yes, those ones are in their issue, so they don't need to be included in this one. 


> If I change the swapping tokens e.g. from DAI to USDC without trading the value which was inserted before remains in the From input.
> I think it might be nicer to clear the sections when choosing a new token

I'm not sure about this one, sometimes your enter the amount and then select the token. It would annoy me if it's removed in some situations. When in doubt of this ones, I would default to trusting uniswap criteria, since they have been collecting feedback a while. If this is an issue at the end, I'm sure we will get it in our own user sessions. 

> Custom slippage%:
I think it might make sense to limit the decimal input only allowing to place two positions after the decimal point.

Probably you are right, but I would also wait for input. 
One thing to note is this app is harder to modify than if it was our app, because we can't touch uniswap code directly and we need to do overrides using a technique that makes changes harder. We can do anything if it's important, but the more we deviate from uniswap the harder the problem it gets (every change adds technical debt to the project).

**Comment by anxolin** [comment]
> In the header it says connect to a wallet and in the body part it says connect wallet

I agree, we should be coherent. Not sure f it has to do with "Responsive" so they cut the messages. I will edit your issue and add a task section with it. 

> In light mode "web and contracts" link and the block link are barely visible

@MareenG We are solving this in the cow app. Currently working doing. Expect it for next version. 

> in the explorer the owl in the header spins if you click on it would be cool to have the same for swap

We killed the OWL. now is wearing a Cow costume

**Comment by MareenG** [comment]
> II'm not sure about this one, sometimes your enter the amount and then select the token. It would annoy me if it's removed in some situations. When in doubt of this ones, I would default to trusting uniswap criteria, since they have been collecting feedback a while. If this is an issue at the end, I'm sure we will get it in our own user sessions.

Agree, clearly sounds like user preference :) 

> Probably you are right, but I would also wait for input.
One thing to note is this app is harder to modify than if it was our app, because we can't touch uniswap code directly and we need to do overrides using a technique that makes changes harder. We can do anything if it's important, but the more we deviate from uniswap the harder the problem it gets (every change adds technical debt to the project).

It is not important. especially of it might result in more problems.
